travis/test_server.py

```diff
@@ -324,7 +326,6 @@ def main(argv=None):
     else:
         print("Modules to test: %s" % tested_addons)
     # setup the base module without running the tests
-    dbtemplate = "openerp_template"
```

Comment:
OK, I agree github reviews are confusing... 😄 They can get you pretty off focus...